adhocteam/Head-Start-TTADP
# TTA-HUB: 397 Add Pill Filters to Grantee Page

## Description of change

Add pill filters to grantee records page.

## How to test

Go to the Grantee Records page and search for any grantee. Select any of the grantee results. Click the TTA History tab and add filters. All added filters should show as pills that reflect the corresponding filter. Removing filters should also remove the filter from the filters drop down.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-397


## Checklists

### Every PR


`- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
